cmd/virt-handler/virt-handler.go

```diff
@@ -334,7 +311,37 @@ func (app *virtHandlerApp) Run() {
 	stop := make(chan struct{})
 	defer close(stop)
 	factory.Start(stop)
-	cache.WaitForCacheSync(stop, factory.ConfigMap().HasSynced, vmiInformer.HasSynced)
+
+	selinuxLauncherType := clusterConfig.GetSELinuxLauncherType()
+	se, exists, err := selinux.NewSELinux()
+	if err == nil && exists {
+		for _, dir := range []string{app.VirtShareDir, app.VirtLibDir} {
+			if labeled, err := se.IsLabeled(dir); err != nil {
+				panic(err)
+			} else if !labeled {
+				err := se.Label("container_file_t", dir)
+				if err != nil {
+					panic(err)
+				}
+			}
+			err := se.Restore(dir)
+			if err != nil {
+				panic(err)
+			}
+		}
+		// Only install KubeVirt's policy if not using a custom one
+		if selinuxLauncherType == "" {
```

Comment:
Maybe compare it to `se.DefaultSELinuxLauncherType`?

Reply:
Thanks! It's always better to use a named value than a magic string--even an empty one.

Reply:
@stu-gott did you want to address something here?